game/script-events.rpy

```diff
@@ -3444,10 +3467,399 @@ label holiday_easter:
+        $ jnPause(2)
+
+        n 5csqcalsbr "..."
+        n 4fnmfllsbl "H-{w=0.2}hey!{w=0.75}{nw}"
+        extend 4fllfllsbl " Come on."
+        n 2cdlbolsbl "Don't give me that look,{w=0.2} [player].{w=0.75}{nw}"
+        extend 2ccsemlsbl " Just..."
+        n 1ksrpulsbl "..."
+        n 1ccsfll "...Just take it already.{w=1}{nw}"
+        extend 4cnmcal " Before I change my mind about the whole thing."
+        n 4cdrcal "..."
+
+        show natsuki 4kdrcal
+        $ jnPause(0.5)
+        call show_poem(halloween_poem)
+        show natsuki 1cnmcal
+
+        n 1cnmajl "You..."
+        n 4cllajl "You did actually read all of that..."
+        n 5csgsll "Right?"
+        n 2ccseml "Because you have no {i}idea{/i} how much of a pain that was at short notice.{w=0.75}{nw}" 
+        extend 2clleml " Especially with you sneaking around all the time.{w=1}{nw}"
+        extend 2ccspol " You dork."
+        n 1clrbolsbl "..."
+        n 4ksrpulsbl "But..."
+        n 4ksgpulsbl "Seriously,{w=0.2} [player]?"
+        n 1kslsllsbl "..."
+
+        show natsuki 4ccsunlsbl at jn_center zorder JN_NATSUKI_ZORDER
+        show black zorder JN_BLACK_ZORDER with Dissolve(0.5)
+        $ jnPause(2)
+        play audio clothing_ruffle
+        $ jnPause(3.5)
+        show natsuki 5cslbolsbr at jn_center zorder JN_NATSUKI_ZORDER
+        hide black with Dissolve(1.25)
+
+        n 5kslpul "...T-{w=0.2}thank you.{w=0.75}{nw}"
+        extend 2knmbol " For being with me for Halloween,{w=0.2} I mean."
+        n 4ccsfll "I know I never got to really dress up,{w=0.75}{nw}"
+        extend 4cdrfll " or make tons of fancy decorations,{w=0.75}{nw}"
+        extend 1ksrsll " or go partying or anything exciting like that."
+        n 1nlrpul "But...{w=1.25}{nw}"
+        extend 5nsrssfsbl " I-{w=0.2}I guess I can settle for treating myself to you instead."
+        n 5nsrajlsbl "So..."
+        n 1csrfslsbl "...Yeah."
+        $ chosen_endearment = jn_utils.getRandomEndearment()
+        n 4cchsmlsbl "H-{w=0.2}happy Halloween,{w=0.2} [chosen_endearment]."
+        n 4cslsmlsbl "..."
+        n 3ccsfllsbr "N-{w=0.2}now,{w=0.2} enough of all the lovey-dovey stuff.{w=0.75}{nw}"
+        extend 3cllfllsbr " Jeez,{w=0.2} [player]."
+        n 3csqpo "Have you {i}totally{/i} forgotten what today is meant to be for already or what?"
+        n 7fsqsm "..."
+        n 7fsqss "No?"
+        n 3fcsss "Heh.{w=0.75}{nw}"
+        extend 3ccsbg " Then you better start preparing,{w=0.2} [player]."
+        n 4cslbgsbr "Being here with you might be a treat..."
+        $ jn_stickers.stickerWindowPeekUp(at_right=True)
+        n 4fsqbs "...But you bet I've still got a ton of {i}tricks{/i} up my sleeve!{w=0.75}{nw}"
+        extend 2nchgn " Ehehe."
+
+        show black zorder JN_BLACK_ZORDER with Dissolve(0.5)
+        $ jnPause(1)
+        play audio drawer
+        $ Natsuki.clearDeskItem(jn_desk_items.JNDeskSlots.centre)
```

Comment:
For some reason the desk item didn't clear for me at all and stayed after the event as well.

Reply:
Which desk item? The book and pumpkin are meant to stay, only the poem should clear at this point

Reply:
Everything stayed, including the poem.

Reply:
This should be resolved